apps/api/src/bootstrap.ts

```diff
@@ -121,7 +122,8 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
   await setupSwagger(app);
 
   Logger.log('BOOTSTRAPPED SUCCESSFULLY');
-
+  app.useGlobalFilters(new AllExceptionsFilter(app.get(PinoLogger)));
```

Comment:
@tatarco The PR looks great! I expected to add this but remove the current handling of exceptions and erroneous response bodies. Where are we generating response 5xx and 4xx responses today? 

Do we need to shortcircuit that or remove the code alltogether?

Reply:
Richard and I did some research, this does not damage the existing functionality, I will attempt to use the sentry ID if possible, currently we are using the default simplistic node.js error mapping which is just mapping 500 for unexpected exceptions and map other apiError's with their constructor values.